inst/components/scss/sidebar.scss

```
@@ -318,6 +325,20 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
         left: unset;
       }
 
+      .sidebar {
```

Not sure if this is important or not, but we seem to do this consistently elsewhere?

```suggestion
      > .sidebar {
```

---

Good catch, yes we need the `>`